R_code_&_data/titanic.R

```diff
@@ -42,4 +43,4 @@ install.packages( arules , scatterplot3d, vcd, seriation, igraph,"grid","cluster
 install.packages("arulesViz")
 
 library(arulesViz)
-plot(rules.pruned)
```

Comment:
im not sure what you are trying to change here. there is no difference between the line you took out and the line you out back in. So i won't be accepting this one.